src/style.css

```diff
@@ -1 +1,59 @@
 /* Styles go here */
+body {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", Arial, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.nav {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+  list-style: none;
+  padding: 23px 0;
```

Comment:
You already set same paddings to link, so no need to duplicate, container will het this child height (but your links should be blocks or inline-blocks, vertical paddings works incorrect for inline elements)

Reply:
I need padding in any case, because the indents for the link increase the clickable area, and the indents for the list are needed so that the elements do not stick to the top of the page. Thank you, I will take it into account, but I tried to fix it, but it didn't work for me. I must be doing something wrong.

Reply:
Actually you don't need them 🙂 As I said you need to set `display: block` to links

Reply:
https://user-images.githubusercontent.com/84275496/214909096-e63b71f9-2e2b-4f50-92cf-f3e51a971edb.mov

Reply:
You can leave it as it is now, it's not critical, but keep in mind that vertical margins and paddings works incorrect for inline elements

Reply:
Thank you, I understood what my mistake was.